core/src/main/java/io/kestra/plugin/core/flow/Subflow.java

```diff
@@ -296,18 +296,18 @@ public SubflowId subflowId() {
     @Getter
     public static class Output implements io.kestra.core.models.tasks.Output {
         @Schema(
-            title = "The ID of the subflow execution."
+            title = "The subflow execution ID"
         )
         private final String executionId;
 
         @Schema(
-            title = "The final state of the subflow execution.",
+            title = "The subflow execution final state",
```

Comment:
```suggestion
            title = "The final state of the subflow execution",
```

I think it was clearer before